docs/reference/announcements-release-notes/850/850-announcements.md

```diff
@@ -1,39 +1,36 @@
 ---
-id: announcements-850
+id: 850-announcements
 title: "8.5 - 8.3 Announcements"
-description: "Important announcements including deprecation & removal notices for the Camunda 8.5, 8.4, 8.3, and 8.2 releases."
+description: "Supported environment changes and breaking changes or deprecations for the Camunda 8.5, 8.4, and 8.3 releases."
+toc_max_heading_level: 2
 ---
 
-Important changes and updates for the Camunda 8.5, 8.4, and 8.3 releases are summarized below.
+Supported environment changes and breaking changes or deprecations for the Camunda 8.5, 8.4, and 8.3 releases are summarized below.
 
 ## Camunda 8.5
 
-| Release date | End of maintenance | Release notes                                        |
-| :----------- | :----------------- | :--------------------------------------------------- |
-| 9 April 2024 | 14 October 2025    | [8.5 release notes](/reference/release-notes/850.md) |
+| Release date | End of maintenance | Release notes                                                                        |
+| :----------- | :----------------- | :----------------------------------------------------------------------------------- |
+| 9 April 2024 | 14 October 2025    | [8.5 release notes](/reference/announcements-release-notes/850/850-release-notes.md) |
 
-### Updated SaaS URLs
+### Supported environment changes
 
-We will simplify the URL for Camunda 8 SaaS from cloud.camunda.io ([console.cloud.camunda.io](https://console.cloud.camunda.io/)) to camunda.io ([console.camunda.io](http://console.camunda.io/)).
+#### Zeebe Go client
 
-On or around July 9th, users will be directed to the new URLs. Both URLs will continue to be active for at least 18 months so navigation from supported versions of components like Operate is still possible.
-
-Internal allowlisting or active rules for [cloud.camunda.io](http://cloud.camunda.io/) must be transitioned to the new [camunda.io](http://camunda.io/) URL. This change primarily affects Console and Modeler. During sign up, users will be briefly redirected through [accounts.cloud.camunda.io](http://accounts.camunda.io/), which will also be updated.
+- Raised minimum Go version to 1.21 for the Zeebe Go client
```

Comment:
This Go version is only noted in 8.5 -- not sure if this changes every release?

Reply:
Doesn't matter 8.6 and on because it's community-supported now and no longer in the docs 😜